config/run/kustomization.yaml

```diff
@@ -0,0 +1,10 @@
+# Value of this field is prepended to the
+# names of all resources, e.g. a deployment named
+# "wordpress" becomes "alices-wordpress".
+# Note that it should also match with the prefix (text before '-') of the namespace
+# field above.
```

Comment:
can we make this comment more relevant to the purpose of this file?

Reply:
@chatton this new kustomization file is for when we want to run the `manager` in localhost instead of a K8s cluster right?

Reply:
yes, this is for when we don't have a k8s deployment running anywhere, the operator runs as a local go binary. We just create the required roles/serviceaccount for everything to run correctly.

Reply:
done